save-core/build.gradle.kts

```diff
@@ -77,6 +79,17 @@ tasks.withType<KotlinCompile<*>>().forEach {
     it.dependsOn(generateVersionFileTaskProvider)
 }
 
+// Integration test should be able to have access to binary during the execution
+val os = getCurrentOperatingSystem()
+tasks.getByName("jvmTest").dependsOn(
```

Comment:
Can this task be moved to save-cli project? Currently you get a circular dependency: save-cli depends on save-core for compilation, but save-core depends on save-cli for tests.

Reply:
Initially I create this test in `save-cli`, but then I thought, that it is not really good, because in this case in `save-cli` we should also download the `diktat` and `ktlint` and then remove them (by tasks from `save-core`). 

So, since we run tests in parallel, it could lead to some problems: `save-cli` will download resources for it's tests and in the same time `save-core` could delete them, because integration tests from `save-core` could be already finished.

Also I there were some other problems which I didn't remember. However now they could be not actual, and I can try to move this test to `save-cli` again

Reply:
Will wait your opinion about my comment above before changing

Reply:
Yeah, that's not a very good structure anyway. I'd suggest to move this test to save-cli and add dependency on `save-cli:jvmTest` to the task, that cleans up downloaded files.